src/js/pensions/config/form.js

```diff
@@ -493,6 +531,202 @@ const formConfig = {
               }
             }
           }
+        },
+        spouseInfo: {
+          title: 'Spouse information',
+          path: 'household/spouse-info',
+          depends: isMarried,
+          uiSchema: {
+            'ui:title': 'Spouse information',
+            spouseDateOfBirth: _.merge(currentOrPastDateUI(''), {
+              'ui:options': {
+                updateSchema: createSpouseLabelSelector(spouseName =>
+                  `${spouseName.first} ${spouseName.last}’s date of birth`)
+              }
+            }),
+            spouseSocialSecurityNumber: _.merge(ssnUI, {
+              'ui:title': '',
+              'ui:options': {
+                updateSchema: createSpouseLabelSelector(spouseName =>
+                  `${spouseName.first} ${spouseName.last}’s Social Security number`)
+              }
+            }),
+            spouseIsVeteran: {
+              'ui:widget': 'yesNo',
+              'ui:options': {
+                updateSchema: createSpouseLabelSelector(spouseName =>
+                  `Is ${spouseName.first} ${spouseName.last} also a Veteran?`)
+              }
+            },
+            spouseVaFileNumber: {
+              'ui:title': 'What is their File Number?',
+              'ui:options': {
+                expandUnder: 'spouseIsVeteran'
+              },
+              'ui:required': form => form.spouseIsVeteran === true,
+              'ui:errorMessages': {
+                pattern: 'File number must be 8 digits'
+              }
+            },
+            liveWithSpouse: {
+              'ui:widget': 'yesNo',
+              'ui:options': {
+                updateSchema: createSpouseLabelSelector(spouseName =>
+                  `Do you live with ${spouseName.first} ${spouseName.last}?`)
+              }
+            },
+            spouseAddress: _.merge(address.uiSchema('Spouse address', false, form => form.liveWithSpouse === false),
+              {
+                'ui:options': {
+                  expandUnder: 'liveWithSpouse',
+                  expandUnderCondition: false
+                }
+              }
+            ),
+            reasonForNotLivingWithSpouse: {
+              'ui:title': 'What is the reason you do not live with your spouse?',
+              'ui:required': form => form.liveWithSpouse === false,
+              'ui:options': {
+                expandUnder: 'liveWithSpouse',
+                expandUnderCondition: false
+              }
+            },
+            monthlySpousePayment: {
+              // TODO: bold contribute monthly
+              'ui:title': 'How much do you contribute monthly to your spouse’s support?',
```

Comment:
Yes, I just either needed to change the extension here or put it in the helper file, which I didn't do on the first pass. I'll update it.